README.md

```diff
@@ -18,13 +18,32 @@ The operator image needs to be pushed to a remote repository:
 ```shell script
 podman push quay.io/<insert username>/wmco:$VERSION_TAG
 ```
-
-## Testing locally
+## Development workflow
+To run the operator on a cluster (for testing purpose only).
 To run the e2e tests for WMCO locally against an OpenShift cluster set up on AWS, we need to setup the following environment variables.
```

Comment:
These two sentences does not flow correctly. Please look at how it appears on the README: `To run the operator on a cluster (for testing purpose only). To run the e2e tests for WMCO locally against an OpenShift cluster set up on AWS, we need to setup the following environment variables.`

This should be `To run the operator development purposes or to run the e2e tests, against an OpenShift cluster set up on AWS, we need to setup the following environment variables.` or something along those lines.